templates/tease-taxonomy-action.twig

```diff
@@ -0,0 +1,69 @@
+<li id="result-row-{{ post.ID }}" class="d-flex search-result-list-item">
+	{% include "blocks/image.twig" with { post, dummy_thumbnail } %}
```

Comment:
My bad, this shortcut actually doesn't work in twig templates apparently, the page crashes on my local because of it! Sorryyyyy 😬